xclim/core/options.py

```diff
@@ -71,6 +73,7 @@ def _valid_missing_options(mopts):
     SDBA_ENCODE_CF: lambda opt: isinstance(opt, bool),
     KEEP_ATTRS: _KEEP_ATTRS_OPTIONS.__contains__,
     AS_DATASET: lambda opt: isinstance(opt, bool),
+    MAP_BLOCKS: lambda opt: isinstance(opt, bool),
```

Comment:
is this an option because we're still not sure if the performance is good? Or there will always be situations where no wrapper is preferable?

Reply:
I think there would be situations where no wrapper is preferable, but it's mostly indeed because I'm not sure of the performance. In many cases, it should have a insignificant impact.